app/src/main/res/layout/activity_ascend.xml

```diff
@@ -20,17 +20,16 @@
 <ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
```

Comment:
It seems to be a bug in the Android library because this layout was working with the previous implementation but with view binding the margins were not recognized anymore.